frontend/public/components/masthead-toolbar.jsx

```diff
@@ -216,7 +216,7 @@ class MastheadToolbarContents_ extends React.Component {
 
   _getAdditionalLinks(links, type) {
     return _.sortBy(
-      _.filter(links, (link) => link.spec.location === type),
+      _.filter(links, (link) => link.spec.location === type && link.metadata.name !== ACM_LINK_ID),
```

Comment:
Worth adding a comment here explaining why we're not showing the ACM link.

Reply:
> I suggest we open a bug to track removing the [cy.]wait if possible.

I was overthinking the cypress test I created for the new url query params.  The test runs succesfully even if Guided Tour modal is open.  For now I just removed the `cy.wait()`.   Long term I think we need to use [app.waitForLoad()](https://github.com/openshift/console/blob/1ef2f7e24430a74f945b6990270dedbcd507dada/frontend/packages/dev-console/integration-tests/support/pages/app.ts#L14) and switch perspectives & close Guided Tour like dev console [does here](https://github.com/openshift/console/blob/1ef2f7e24430a74f945b6990270dedbcd507dada/frontend/packages/dev-console/integration-tests/support/pages/app.ts#L29).

> Worth adding a comment

...added

Reply:
> The test runs succesfully even if Guided Tour modal is open.

Guided Tour modal continues to cause failures :disappointed: 
![image](https://user-images.githubusercontent.com/12733153/116711805-33463500-a9a1-11eb-88f5-bd3ddbd76d14.png)
![image](https://user-images.githubusercontent.com/12733153/116711831-39d4ac80-a9a1-11eb-9273-d059f32e5807.png)

Adding a WIP while I investigate...